templates/README.md

```diff
@@ -20,64 +20,77 @@ Only the structure of the project file is relevant to the template. The actual
 list of source code and targets isn't.
 
 We currently have template files for GNU Make, Visual Studio 2010 to 2015,
-and Bazel. In the future, we would like to expand to generating gyp or cmake
+and [Bazel](http://bazel.io). In the future, we would like to expand to
+generating [gyp](https://gyp.gsrc.io/) or [cmake](https://cmake.org)
```

Comment:
I have a PR open for cmake also.

On Tue, Mar 1, 2016, 4:59 PM Nicolas Noble <notifications@github.com> wrote:

> In templates/README.md
> <https://github.com/grpc/grpc/pull/5524#discussion_r54663951>:
>
> > @@ -20,64 +20,77 @@ Only the structure of the project file is relevant to the template. The actual
> >  list of source code and targets isn't.
> >
> >  We currently have template files for GNU Make, Visual Studio 2010 to 2015,
> > -and Bazel. In the future, we would like to expand to generating gyp or cmake
> > +and [Bazel](http://bazel.io). In the future, we would like to expand to
> > +generating [gyp](https://gyp.gsrc.io/) or [cmake](https://cmake.org)
>
> It's really an epsilon away from being standalone, I'd say. But fair
> enough, it's only for node at the moment.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/5524/files#r54663951>.
>